src/aics-image-viewer/components/ChannelsWidgetRow/index.tsx

```diff
@@ -119,14 +119,19 @@ const ChannelsWidgetRow: React.FC<ChannelsWidgetRowProps> = (props: ChannelsWidg
   );
 
   const renderActions = (): React.ReactNode => (
```

Comment:
Just an FYI, for some reason the contents of the channels widget come before these action buttons in the HTML. It makes tab navigation a little funky... Is that something easily fixable or does it start to break the layout?

Reply:
That's very strange, and seems to be a problem that would require more reaching into ant's structure than it's worth for this issue. Merits an issue ticket though.

Reply:
Opened #246 to track this.